hcl/eval/partial.go

```diff
@@ -1119,6 +1260,70 @@ func isTmFuncall(tok *hclwrite.Token) bool {
 		strings.HasPrefix(string(tok.Bytes), "tm_")
 }
 
+func isSameTokens(a, b hclwrite.Tokens) bool {
```

Comment:
areSameTokens or areEqualTokens or tokensAreEqual ? "is" is singular but we are talking about tokens plural here.